src/Private/OpenApi.ps1

```diff
@@ -48,18 +78,39 @@ function ConvertTo-PodeOAObjectSchema {
         $obj[$type] = @{
             schema = $null
         }
-
+        if ($Array) {
+            $obj[$type].schema = @{
+                'type'  = 'array'
+                'items' = $null
+            }
+        }
         # add a shared component schema reference
         if ($Schemas[$type] -is [string]) {
-            if (!(Test-PodeOAComponentSchema -Name $Schemas[$type])) {
-                throw "The OpenApi component schema doesn't exist: $($Schemas[$type])"
-            }
-
-            $obj[$type].schema = @{
-                '$ref' = "#/components/schemas/$($Schemas[$type])"
+            if (@('string', 'integer' , 'number', 'boolean' ) -contains $Schemas[$type]) {
```

Comment:
same as before STRIng is not a valid type for OpenAPI

Reply:
If you can guarantee by this point that `$Schemas[$type]` as a string will always be lowercase then that's fine. But for safety, in case it can come directly from unmodified user input, using `-icontains` and then `'type' = $Schemas[$type].ToLower()` would work.

If it can come directly from user input, then maybe moving the ToLower to a higher level would work best.

Reply:
The issue is that String with uppercase S is not a valid OpenAPI type
OpenAPI accept only lowercase name as type

Reply:
maybe I can force everything to lowercase. Just in case someone wants to use a hashtable as property

Reply:
I agree, but we shouldn't penalise a user just because they typed `'String'` instead of `'string'` when we can very easily lowercase it for them.

Reply:
I can do it only for type